crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -232,7 +246,7 @@ impl<T: Component> WorldQuery for &T {
     type ReadOnlyFetch = ReadFetch<T>;
 }
 
-/// The [`FetchState`] of `&T`.
+/// The [`FetchState`] of `&T`, when used as a [Component] in a [WorldQuery].
```

Comment:
Is there a difference between `[StructDefinition]` and ```[`StructDefinition`]``` when it comes to what is being linked / how it is rendered in the rust-docs?

Reply:
The latter will be shown in code-style monospace text.

Reply:
Gotcha, thanks!